Plugins/IngameDebugConsole/Scripts/DebugLogConsole.cs

```diff
@@ -197,17 +197,10 @@ public static void SearchAssemblyForConsoleMethods( Assembly assembly )
 		{
 			try
 			{
-				foreach( Type type in assembly.GetExportedTypes() )
+				IEnumerable<ConsoleAttribute> consoleMethods = TypeSearch.GetConsoleMethods(assembly.GetExportedTypes());
+				foreach (ConsoleAttribute consoleMethod in consoleMethods)
 				{
-					foreach( MethodInfo method in type.GetMethods( BindingFlags.Static | BindingFlags.Public | BindingFlags.DeclaredOnly ) )
-					{
-						foreach( object attribute in method.GetCustomAttributes( typeof( ConsoleMethodAttribute ), false ) )
-						{
-							ConsoleMethodAttribute consoleMethod = attribute as ConsoleMethodAttribute;
-							if( consoleMethod != null )
-								AddCommand( consoleMethod.Command, consoleMethod.Description, method, null, consoleMethod.ParameterNames );
-						}
-					}
+					consoleMethod.Load();
```

Comment:
So, can you restore this old code and instead of calling AddCommand, just fill the List? Then we can Sort that list and process its elements.